src/js/formers/KycCorporateFormer.js

```diff
@@ -0,0 +1,140 @@
+import { Former } from './Former'
+import { base, BLOB_TYPES } from '@tokend/js-sdk'
+import { doc, str, reqId } from './op-build-helpers'
+import { KycCorporateRecord } from '@/js/records/entities/kyc-corporate.record'
+import { KycRequestRecord } from '@/js/records/requests/kyc-request.record'
+import { KycRecoveryRequestRecord } from '@/js/records/requests/kyc-recovery-request.record'
+import { BlobRecord } from '@/js/records/entities/blob.record'
+import { uploadDocumentsDeep } from '@/js/helpers/upload-documents'
+import { createPrivateBlob, getCurrentAccId } from '@/js/helpers/api-helpers'
+import { buildKycRecoveryOp } from '@/js/helpers/kyc-helpers'
+import { DocumentContainer } from '@/js/helpers/DocumentContainer'
+import { keyValues } from '@/key-values'
+
+/**
+ * Collects the attributes for kyc-corporate operations
+ * @class
+ * @implements {Former}
+ */
+export class KycCorporateFormer extends Former {
+  _defaultAttrs = {
+    requestId: '0',
+    name: '',
+    company: '',
+    headquarters: '',
+    industry: '',
+    teamSize: 0,
+    website: '',
+    avatar: DocumentContainer.fromObj(),
```

Comment:
Don't we now use `Document` from sdk?

Reply:
`Document` is not merged yet, the changes will be applied as soon as they are merged

Reply:
done